test/test_data/output_formatter.medium.fortio-noquirks.gold

```diff
@@ -0,0 +1,483 @@
+    "End": 0.060055551,
+    "Percent": 30,
+    "Count": 5
+   },
+   {
+    "Start": 0.060055551,
+    "End": 0.062949375,
+    "Percent": 40,
+    "Count": 6
+   },
+   {
+    "Start": 0.062949375,
+    "End": 0.065077247,
+    "Percent": 50,
+    "Count": 5
+   },
+   {
+    "Start": 0.065077247,
+    "End": 0.066666495,
+    "Percent": 55.000000000000007,
+    "Count": 3
+   },
+   {
+    "Start": 0.066666495,
+    "End": 0.067948543,
+    "Percent": 60,
+    "Count": 2
+   },
+   {
+    "Start": 0.067948543,
+    "End": 0.068751359,
+    "Percent": 65,
+    "Count": 3
+   },
+   {
+    "Start": 0.068751359,
+    "End": 0.072949759,
+    "Percent": 70,
+    "Count": 3
+   },
+   {
+    "Start": 0.072949759,
+    "End": 0.074465279,
+    "Percent": 75,
+    "Count": 2
+   },
+   {
+    "Start": 0.074465279,
+    "End": 0.075792383,
+    "Percent": 77.5,
+    "Count": 2
+   },
+   {
+    "Start": 0.075792383,
+    "End": 0.078065663,
+    "Percent": 80,
+    "Count": 1
+   },
+   {
+    "Start": 0.078065663,
+    "End": 0.078168063,
+    "Percent": 82.5,
+    "Count": 1
+   },
+   {
+    "Start": 0.078168063,
+    "End": 0.080400383,
+    "Percent": 85,
+    "Count": 2
+   },
+   {
+    "Start": 0.080400383,
+    "End": 0.082427903,
+    "Percent": 87.5,
+    "Count": 1
+   },
+   {
+    "Start": 0.082427903,
+    "End": 0.085045247,
+    "Percent": 88.75,
+    "Count": 1
+   },
+   {
+    "Start": 0.085045247,
+    "End": 0.085045247,
+    "Percent": 90,
+    "Count": 0
+   },
+   {
+    "Start": 0.085045247,
+    "End": 0.085626879,
+    "Percent": 91.25,
+    "Count": 1
+   },
+   {
+    "Start": 0.085626879,
+    "End": 0.090656767,
+    "Percent": 92.5,
+    "Count": 1
+   },
+   {
+    "Start": 0.090656767,
+    "End": 0.090656767,
+    "Percent": 93.75,
+    "Count": 0
+   },
+   {
+    "Start": 0.090656767,
+    "End": 0.247513087,
+    "Percent": 94.375,
+    "Count": 1
+   },
+   {
+    "Start": 0.247513087,
+    "End": 0.247513087,
+    "Percent": 95,
+    "Count": 0
+   },
+   {
+    "Start": 0.247513087,
+    "End": 0.247513087,
+    "Percent": 95.625,
+    "Count": 0
+   },
+   {
+    "Start": 0.247513087,
+    "End": 0.281362431,
+    "Percent": 96.25,
+    "Count": 1
+   },
+   {
+    "Start": 0.281362431,
+    "End": 0.281362431,
+    "Percent": 96.875,
+    "Count": 0
+   },
+   {
+    "Start": 0.281362431,
+    "End": 0.281362431,
+    "Percent": 97.1875,
+    "Count": 0
+   },
+   {
+    "Start": 0.281362431,
+    "End": 0.281362431,
+    "Percent": 97.5,
+    "Count": 0
+   },
+   {
+    "Start": 0.281362431,
+    "End": 0.281362431,
+    "Percent": 97.8125,
+    "Count": 0
+   },
+   {
+    "Start": 0.281362431,
+    "End": 0.310886399,
+    "Percent": 98.125,
+    "Count": 1
+   },
+   {
+    "Start": 0.310886399,
+    "End": 0.310886399,
+    "Percent": 100,
+    "Count": 0
+   }
+  ],
+  "Min": 0.053798911,
+  "Max": 0.310886399,
+  "Sum": 4.156954618,
+  "Avg": 0.078433106,
+  "StdDev": 0.05052302,
```

Comment:
Could we move these fields (line 197 to line 201) up to under line 8 "Count" field as what fortio has?

Reply:
That is quite hard to pull off in the current approach. Looking at it, the proto json serialiser we use here alphabetically sorts this. I think it can do that, because after parsing the difference usually doesn't matter? Do you see a way that the re-ordering could cause trouble?

Reply:
Not yet. Just noticed the order is mismatch with fortio. Don't think it will cause any trouble.